Urban-Fun/View Controllers/SearchViewController.m

```diff
@@ -251,6 +260,7 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         ActivityDetailsViewController *vc = [segue destinationViewController];
         vc.activity = dataToPass;
         vc.activitydetailsDelegate = self;
+        vc.navigationItem.title = dataToPass.title;
```

Comment:
I previously missed this, but I think we can improve this naming `dataToPass` doesn't give me any clue about the type of data/object I'm passing. As general rule, think about other people reading your code, they need to know what's happening and the naming helps a lot, instead of `dataToPass` you can simply name it `activity` or `activityToPass`